### PR TITLE
Fixes gitpod-io/website#343

### LINKED ISSUE #343: [blog post] title has some unecessary spacing & decenders have white on top.
With some recent changes that we made The title on a Blog post now has some unnecessary space in the top left of first-line:

image

Also, some part of the descenders in the first-line of the title is now under the white background of the 2nd line like the letter **g** in the screenshot below: 
image

before that it was like this:

image

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #343 

This is what it looks like now: 

image

The above change is approved by @ChristinFrohne here https://github.com/gitpod-io/website/issues/343#issuecomment-567500181

Signed-off-by: Nisar Hassan Naqvi <syednisarhassan12@gmail.com>

### CODE REVIEW

File: src/components/PostBanner.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ const StyledPostBanner = styled.header`
         background: ${colors.white};
         box-shadow: ${shadows.light1};
         padding: 1rem 2rem;
+        background: ${colors.white};
```
Comment (svenefftinge) [inline src/components/PostBanner.tsx]: the same is three lines above?

Comment (nisarhassan12) [reply src/components/PostBanner.tsx]: Oops! sorry stupid me. This is now being removed.

Comment (nisarhassan12) [reply src/components/PostBanner.tsx]: fixed it. Thanks!

**Comment by svenefftinge** [comment]
doesn't look good here: https://deploy-preview-353--gitpod-staging.netlify.com/blog/local-services-in-gitpod/

**Comment by svenefftinge** [comment]
and here: https://deploy-preview-353--gitpod-staging.netlify.com/blog/status-page/

**Comment by nisarhassan12** [comment]
@svenefftinge  I have updated the branch! Does this look good?
image

image

**Comment by svenefftinge** [comment]
Hey Nisar, sorry I think we missunderstood. I liked the styling but in the pictures I shared there were empty boxes because of missing subtitles. I have fixed the issue and pushed to master.

**Pull request closed without merging**